Sources/SwiftUIKit/Views/Map.swift

```diff
@@ -0,0 +1,453 @@
+    super.showAnnotations(annotations, animated: animated)
+    
+    return self
+  }
+  
+  @discardableResult
+  public func showAnnotations(_ annotations: MKAnnotation..., animated: Bool = true) -> Self {
+    super.showAnnotations(annotations, animated: animated)
+    
+    return self
+  }
+}
+
+// MARK: - Constraining the Map View
+
+@available(iOS 13.0, *)
+extension Map {
+  @discardableResult
+  public func cameraBoundary(_ boundary: MKMapView.CameraBoundary?, animated: Bool = true) -> Self {
+    setCameraBoundary(boundary, animated: animated)
+    
+    return self
+  }
+  
+  @discardableResult
+  public func setCameraZoomRange(_ cameraZoomRange: MKMapView.CameraZoomRange?, animated: Bool) -> Self {
+    super.setCameraZoomRange(cameraZoomRange, animated: animated)
+    
+    return self
+  }
+}
+
+// MARK: - Configuring the Map's Appearance
+extension Map {
+  @discardableResult
+  public func camera(_ camera: MKMapCamera, animated: Bool = true) -> Self {
+    setCamera(camera, animated: animated)
+    
+    return self
+  }
+  
+  @discardableResult
+  public func showBuildings(_ bool: Bool? = nil) -> Self {
+    showsBuildings = bool ?? !showsBuildings
+    
+    return self
+  }
+}
+
+@available(iOS 13.0, *)
+extension Map {
+  @discardableResult
+  public func showCompass(_ bool: Bool? = nil) -> Self {
+    showsCompass = bool ?? !showsCompass
+    
+    return self
+  }
+  
+  @discardableResult
+  public func showScale(_ bool: Bool? = nil) -> Self {
+    showsScale = bool ?? !showsScale
+    
+    return self
+  }
+  
+  @discardableResult
+  public func showTraffic(_ bool: Bool? = nil) -> Self {
+    showsTraffic = bool ?? !showsTraffic
+    
+    return self
+  }
+  
+  @discardableResult
+  public func pointOfInterestFilter(_ filter: MKPointOfInterestFilter?) -> Self {
+    pointOfInterestFilter = filter
+    
+    return self
+  }
+}
+
+// MARK: - Displaying the User's Location
+extension Map {
+  @discardableResult
+  public func showUserLocation(_ bool: Bool? = nil) -> Self {
+    showsUserLocation = bool ?? !showsUserLocation
+    
+    return self
+  }
+  
+  @discardableResult
+  public func userTrackingMode(_ mode: MKUserTrackingMode, animated: Bool = true) -> Self {
+    setUserTrackingMode(mode, animated: animated)
+    
+    return self
+  }
+}
+
+// MARK: - Managing Annotation Selections
+extension Map {
+  @discardableResult
+  public func select(_ annotation: MKAnnotation, animated: Bool = true) -> Self {
+    selectAnnotation(annotation, animated: animated)
+    
+    return self
+  }
+  
+  @discardableResult
+  public func deselect(_ annotation: MKAnnotation, animated: Bool = true) -> Self {
+    deselectAnnotation(annotation, animated: animated)
+    
+    return self
+  }
+}
+
+// MARK: - Annotating the Map
+extension Map {
+  @discardableResult
+  public func remove(_ annotations: MKAnnotation...) -> Self {
+    removeAnnotations(annotations)
+    
+    return self
+  }
```

Comment:
I'd prefer we have two functions instead of the variadic... 

```
public func remove(annotation: MKAnnotation) -> Self
public func remove(annotations: [MKAnnotation]) -> Self
```